File: src/main/java/tech/zolhungaj/amqapi/servercommands/globalstate/AllOnlineUsers.java
package tech.zolhungaj.amqapi.servercommands.globalstate;

import com.squareup.moshi.Json;
import tech.zolhungaj.amqapi.servercommands.CommandType;

import java.util.List;

@CommandType("all online users")
public record AllOnlineUsers(
        @Json(name = "array")
        List<String> list
){}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/globalstate/AllOnlineUsers.kt
package tech.zolhungaj.amqapi.servercommands.globalstate

import com.squareup.moshi.Json
import tech.zolhungaj.amqapi.servercommands.CommandType

@JvmRecord
@CommandType("all online users")
data class AllOnlineUsers(
    @Json(name = "array") val list: List<String>
)
